tests/integration/targets/nd_site/tasks/main.yml

```diff
@@ -0,0 +1,246 @@
+# Test code for the ND modules
```

Comment:
Was this tested against an inventory having NDFC, DCNM and Cloud APIC IP addresses?